pyuploadcare/dj/templates/uploadcare/forms/widgets/file.html

```diff
@@ -0,0 +1,45 @@
+{% if value %}
+    <script>
+        window.addEventListener('DOMContentLoaded', (e) => {
+            const uploaderCtx = document.querySelector('#uploaderctx_{{ name }}');
+            {% if config.multiple == "true" %}
+                {# FileGroup #}
+                {% for file in value %}
+                    uploaderCtx.addFileFromUuid("{{ file.uuid }}")
```

Comment:
@nd0ut is there a way to add an existing file group from UUID instead of iterating over its individual files?